src/config/statsd.ts

```diff
@@ -55,27 +55,33 @@ export const elapsedTimeMetrics = (
 ) => {
 	const elapsedTimeInMs = hrtimer();
 	const method = req.method;
+	(req.log || logger).debug({
+		method: req.method,
+		path: req.path,
+		body: JSON.stringify(req.body),
+		query: req.query
+	}, `${method} request initialized for path ${req.path}`);
```

Comment:
Added this as an easier way to debug what's getting hit.  It's been very useful to finding issues quickly.